source/help/messaging/formatting-text.rst

```diff
@@ -235,3 +235,19 @@ Create a table by placing a dashed line under the header row and separating the
 Renders as:
 
 .. image:: ../../images/markdownTable1.PNG
+
+
+Math Formulas
```

Comment:
Propose moving this as a sub section of `Code Block` under the `Synax Highlighting` section.